Source/WebCore/css/CSSProperties.json

```diff
@@ -8834,7 +8834,10 @@
                 "all"
             ],
             "codegen-properties": {
-                "parser-grammar": "<<values>>"
+                "parser-grammar": "<<values>>",
+                 "aliases": [
+                    "user-select"
+                ]
```

Comment:
Unprefixing user-select is a whole other issue that requires more changes. Please check https://bugs.webkit.org/showdependencytree.cgi?id=208677&hide_resolved=1 for a list of bugs to fix before unprefixing user-select

Reply:
I ended up getting web compatibility issues, when I did that 3 years ago, simply because the `user-select` requires more changes.